capsules/extra/src/nmea_i2c.rs

```diff
@@ -0,0 +1,183 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
+
+//! A capsule for I2C NMEA. This is used to read NMEA sentences from a GPS/GNSS
+//! device connected via I2C.
```

Comment:
To get this to a mergeable state, it seems much easier to convince everyone that it's a working driver for a _specific_ GNSS device, rather than a generic NMEA I2C driver. So why don't we rename this file to be an interface for whatever _specific_ chip you're working with?